build.gradle

```diff
@@ -50,40 +54,56 @@ configurations {
 
 dependencies {
 	enigmaRuntime "net.fabricmc:stitch:0.1.2.49+"
-	enigmaRuntime "cuchaz:enigma:0.13.1.99:all"
+
+	// not transitive
+	enigmaRuntime name: "darcula", version: "1.0.0"
+	enigmaRuntime 'de.sciss:syntaxpane:1.2.+'
+	enigmaRuntime 'me.xdrop:fuzzywuzzy:1.2.0'
+	enigmaRuntime "org.bitbucket.mstrobel:procyon-compilertools:0.5.33.7"
+	//enigmaRuntime 'net.fabricmc:procyon-fabric-compilertools:0.5.33-SNAPSHOT'
+	// Uncomment when Rune fixed it
+	enigmaRuntime("cuchaz:enigma:0.13.1.114+") {
```

Comment:
Here, you have the enigma version in 2 places, it should only be defined once

Reply:
Is it even possible to use project properties in buildscript block? I need to see